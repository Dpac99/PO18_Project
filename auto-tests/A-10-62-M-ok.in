100011
1
e1.obj
6
3
D99
P1
3
D1
P0
4
D23
P3
3
D1
PP
4
dd
p1
5
dd
22
0
0
