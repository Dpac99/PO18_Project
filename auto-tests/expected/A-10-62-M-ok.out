Identificador: Menu principal
1 - Abrir
2 - Guardar
3 - Portal Pessoal
5 - Portal do Estudante
6 - Portal do Delegado
0 - Sair
Escolha uma opção: Ficheiro a abrir: Pode preencher inquérito do projecto P2 da disciplina D1
Pode preencher inquérito do projecto P4 da disciplina D1
Menu principal
1 - Abrir
2 - Guardar
3 - Portal Pessoal
5 - Portal do Estudante
6 - Portal do Delegado
0 - Sair
Escolha uma opção: Portal do Delegado
1 - Criar inquérito
2 - Cancelar inquérito
3 - Abrir inquérito
4 - Fechar inquérito
5 - Finalizar inquérito
6 - Mostrar inquéritos de uma disciplina
0 - Sair
Escolha uma opção: Nome da disciplina: Nome do projecto: Abrir inquérito: Operação inválida: A disciplina 'D99' não existe ou não é válida para o utilizador em causa.
Portal do Delegado
1 - Criar inquérito
2 - Cancelar inquérito
3 - Abrir inquérito
4 - Fechar inquérito
5 - Finalizar inquérito
6 - Mostrar inquéritos de uma disciplina
0 - Sair
Escolha uma opção: Nome da disciplina: Nome do projecto: Abrir inquérito: Operação inválida: O projecto 'P0' não existe na disciplina 'D1'.
Portal do Delegado
1 - Criar inquérito
2 - Cancelar inquérito
3 - Abrir inquérito
4 - Fechar inquérito
5 - Finalizar inquérito
6 - Mostrar inquéritos de uma disciplina
0 - Sair
Escolha uma opção: Nome da disciplina: Nome do projecto: Fechar inquérito: Operação inválida: A disciplina 'D23' não existe ou não é válida para o utilizador em causa.
Portal do Delegado
1 - Criar inquérito
2 - Cancelar inquérito
3 - Abrir inquérito
4 - Fechar inquérito
5 - Finalizar inquérito
6 - Mostrar inquéritos de uma disciplina
0 - Sair
Escolha uma opção: Nome da disciplina: Nome do projecto: Abrir inquérito: Operação inválida: O projecto 'PP' não existe na disciplina 'D1'.
Portal do Delegado
1 - Criar inquérito
2 - Cancelar inquérito
3 - Abrir inquérito
4 - Fechar inquérito
5 - Finalizar inquérito
6 - Mostrar inquéritos de uma disciplina
0 - Sair
Escolha uma opção: Nome da disciplina: Nome do projecto: Fechar inquérito: Operação inválida: A disciplina 'dd' não existe ou não é válida para o utilizador em causa.
Portal do Delegado
1 - Criar inquérito
2 - Cancelar inquérito
3 - Abrir inquérito
4 - Fechar inquérito
5 - Finalizar inquérito
6 - Mostrar inquéritos de uma disciplina
0 - Sair
Escolha uma opção: Nome da disciplina: Nome do projecto: Finalizar inquérito: Operação inválida: A disciplina 'dd' não existe ou não é válida para o utilizador em causa.
Portal do Delegado
1 - Criar inquérito
2 - Cancelar inquérito
3 - Abrir inquérito
4 - Fechar inquérito
5 - Finalizar inquérito
6 - Mostrar inquéritos de uma disciplina
0 - Sair
Escolha uma opção: Menu principal
1 - Abrir
2 - Guardar
3 - Portal Pessoal
5 - Portal do Estudante
6 - Portal do Delegado
0 - Sair
Escolha uma opção: 